locale/core.yml

```diff
@@ -148,6 +148,7 @@ core:
       start_discussions_label: Start discussions
       view_discussions_label: View discussions
       view_post_ips_label: View post IP addresses
+      view_user_list_label: View User List
```

Comment:
Oops! sorry about that. For some reason I was thinking in heading form :P
Fixed!